charts/jaeger/values.yaml

```diff
@@ -11,6 +11,12 @@ tag: 1.16.0
 nameOverride: ""
 fullnameOverride: ""
 
+# Security Context for deployments
+securityContext:
```

Comment:
Can we have  both podSecurityContext as well as container level security context? As done in `helm create`: https://github.com/helm/helm/blob/a963736f6675e972448bf7a5fd141628fd0ae4df/pkg/chartutil/create.go#L112-L121